tsconfig.json

```diff
@@ -13,7 +13,7 @@
         "strict": true,
         "strictNullChecks": false,
         "stripInternal": true,
-        "target": "es5"
+        "target": "es2015"
```

Comment:
Can you keep the target as ES5? But set the `lib` as the following:

```json"lib": [
    "dom",
    "dom.iterable",
    "es6",
    "scripthost"
],
```

Because you still want TS to compile to ES5.